<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>402</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>402</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>402</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Fan Control Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="application.qrc">
    <normaloff>:/res/fan.png</normaloff>:/res/fan.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QGroupBox" name="fansGroup">
     <property name="title">
      <string>Fan Control Mode</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <property name="horizontalSpacing">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="fansLeftLabel">
        <property name="text">
         <string>Left:</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="fansLeftLayout">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QRadioButton" name="fansLeftCpuRadio">
          <property name="text">
           <string>CPU</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">fanLeftRadioGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="fansLeftGpuRadio">
          <property name="text">
           <string>GPU</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">fanLeftRadioGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="fansLeftMaxRadio">
          <property name="text">
           <string>Maximum</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">fanLeftRadioGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="fansRightLabel">
        <property name="text">
         <string>Right:</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="fansRightLayout">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QRadioButton" name="fansRightCpuRadio">
          <property name="text">
           <string>CPU</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">fanRightButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="fansRightGpuRadio">
          <property name="text">
           <string>GPU</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">fanRightButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="fansRightMaxRadio">
          <property name="text">
           <string>Maximum</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">fanRightButtonGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="tempModeGroup">
     <property name="title">
      <string>Temperature Scale</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QRadioButton" name="celsiusRadio">
        <property name="text">
         <string>Celsius (°C)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">tempModeRadioGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="fahrenheitRadio">
        <property name="text">
         <string>Fahrenheit (°F)</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">tempModeRadioGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="autoTempGroup">
     <property name="title">
      <string>Auto Temperature Range</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="autoTempLayout">
        <item>
         <widget class="QLabel" name="autoTempMin">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QxtSpanSlider" name="autoTempSlider">
          <property name="minimum">
           <number>30</number>
          </property>
          <property name="maximum">
           <number>90</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="autoTempMax">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="autoTempInfo">
        <property name="text">
         <string>This represents the minimum and maximum temperatures which correspond with the range that you select in auto mode. </string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="heatWarningsGroup">
     <property name="title">
      <string>Heat Warnings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="heatWarningsEnabledBox">
        <property name="text">
         <string>Enabled</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="heatWarningsHorizontalLayout">
        <item>
         <layout class="QGridLayout" name="heatWarningsGridLayout">
          <property name="horizontalSpacing">
           <number>9</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="heatWarningsIntervalLabel">
            <property name="text">
             <string>Interval:</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="heatWarningsIntervalSpin">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="minimum">
             <number>15</number>
            </property>
            <property name="maximum">
             <number>7200</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>60</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="heatWarningsIntervalUnit">
            <property name="text">
             <string>seconds</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="heatWarningsThresholdLabel">
            <property name="text">
             <string>Threshold:</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="heatWarningsThresholdSpin">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="minimum">
             <number>50</number>
            </property>
            <property name="maximum">
             <number>95</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>85</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="heatWarningsThresholdUnit">
            <property name="text">
             <string>° C</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="heatWarningsGridPusher">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QxtSpanSlider</class>
   <extends>QSlider</extends>
   <header>qxtspanslider.h</header>
   <slots>
    <signal>spanChanged(int,int)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="tempModeRadioGroup"/>
  <buttongroup name="fanRightButtonGroup"/>
  <buttongroup name="fanLeftRadioGroup"/>
 </buttongroups>
</ui>
